frontend/ui/otamanager.lua

```diff
@@ -175,9 +176,43 @@ function OTAManager:fetchAndProcessUpdate()
                         if self.can_pretty_print then
                             os.execute("./fbink -q -y -7 -pm ' '  ' '")
                         end
-                        UIManager:show(ConfirmBox:new{
-                            text = _("Error updating KOReader. Would you like to delete temporary files?"),
-                            ok_callback = function()
+                        UIManager:show(MultiConfirmBox:new{
+                            text = _("Failed to update KOReader.\n\nYou can:\nCancel, keeping temporary files intact.\nRetry the update process but this time, with a full download.\nAbort and cleanup all temporary files."),
```

Comment:
It can probably be slightly shorter without losing anything:
```suggestion
                            text = _("Failed to update KOReader.\n\nYou can:\nCancel, keeping temporary files.\nRetry the update process with a full download.\nAbort and cleanup all temporary files."),
```